department-of-veterans-affairs/caseflow
# Decision Review Disposition Page Enhancements

Resolves [APPEALS-13693](https://vajira.max.gov/browse/APPEALS-13693)

### Description
This improves the interface of the Decision Review Disposition page.

### Acceptance Criteria
- [x] The Disposition dropdown should be placed above the "Decision description" textbox
    - [x] The dropdown should have a label with the following text above its top-left corner: "Disposition".
        - [x] This label should be bolded.
    - [x] The dropdown should have a label with the following text above its top-right corner: "Required".
        - [x] This label should also be bolded.
    - [x] The placeholder text should be changed to the following: "Select or enter...".
- [x] The "Decision description" textarea's "Optional" label should be over its top-right corner, and not off to the right of the text box.
- [x] This label should be bolded.
- [x] The existing "Decision description" label should be retained (feel free to alter how its set up in the component, however).
    - [x] This label should be bolded.
- [x] The "Complete" button should remain disabled until the following fields have been populated:
    - [x] Disposition dropdown
    - [x] Decision date selector

### Testing Plan
1. Log in as the user 'ACBAUERVVHAH'
2. Navigate to /decision_reviews/vha and click on one of the claimants
3. Observe the new layout
4. The complete button should stay disabled until both a disposition and date are selected.

### User Facing Changes

 BEFORE
---
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/17840614/213786774-227a7b36-d265-4899-9079-df6e6d700a16.png">
AFTER
---
<img width="1097" alt="image" src="https://user-images.githubusercontent.com/17840614/213786628-4dc9536c-c36e-4ffe-958d-555e2d48c406.png">
